app/screens/settings/notifications/notifications.test.tsx

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {renderWithEverything} from '@test/intl-test-helper';
+import TestHelper from '@test/test_helper';
+
+import Notifications from './notifications';
+
+import type UserModel from '@database/models/server/user';
+import type Database from '@nozbe/watermelondb/Database';
+
+jest.mock('react-native-notifications', () => ({
+    Notifications: {
+        isRegisteredForRemoteNotifications: jest.fn(),
+    },
+}));
```

Comment:
We are already mocking `react-native-notifications` on the `setup.ts`. Do you mind adding this mock there?